app/controllers/movies_controller.rb

```diff
@@ -31,5 +33,27 @@ def index
     end
   end
 
-  def show; end
+  def show
+    conn = Faraday.new(
+      url: 'https://api.themoviedb.org/3/movie',
+      params: {api_key: ENV['viewing_party_key'] }
+    )
+
+    response = conn.get("/3/movie/#{params[:id]}")
```

Comment:
I think this is a great first step towards getting our API calls up and working but before finalizing, we should try to make some helper methods to clean up the code a little.